server/testdriver/core/src/main/java/org/infinispan/server/test/core/ContainerInfinispanServerDriver.java

```diff
@@ -230,6 +230,32 @@ private void configureSite(List<String> args) {
       args.add("-Djgroups.cluster.mcast_port=" + configuration.siteDiscoveryPort());
    }
 
+   /*
+    * Once you built the server, you can try to start it and it will create files under the data directory.
+    * Later, you decided to run some tests and it will take ages to find out that the tests are broken because there
+    * are files under the data directory.
+    */
+   private Path validateServerOutput(Path serverOutputPath) {
```

Comment:
We can replace this method with a call to `org.infinispan.commons.Util#recursiveFileRemove`